cogs5e/models/sheet/base.py

```diff
@@ -0,0 +1,284 @@
+from utils.constants import SAVE_NAMES, SKILL_NAMES, STAT_ABBREVIATIONS, STAT_NAMES, SKILL_MAP
+from utils.functions import camel_to_title, verbose_stat
+
+
+class BaseStats:
```

Comment:
Added lots of core classes that all 5e creatures have - the thought is to create some form of unified Sheet/StatBlock system to bring player characters and monsters closer together in the future